### ISSUE TITLE
Limit on played games fetched changed to 200 from 500 

### ISSUE THREAD

**Issue by biggydbs** [isFakeAccount/psnawp#43]
psn fetch_played_games error : error: {"error":{"reason":"BadRequest","source":"Gamelist","code":3239940,"message":"invalid request, requested limit=500 allowed limit=200","referenceId":"2e1e8a09-6331-1031-8522-d58c84c821b7"}}

**Comment by isFakeAccount** [comment]
Pretty sure I adjusted this once already. I am not sure if the best strategy is to change the page size again.

**Comment by isFakeAccount** [comment]
Issue resolved in 1.3.0. Also, in the future if the limit changes, you should be able to change the page size easily. 

```py
title_stat_iterator = user_example.title_stats(limit=10, page_size=10)
```
